client/themes/zenburn.css

```diff
@@ -173,9 +173,9 @@ body {
 }
 
 #form .input {
-	background-color: #434443;
-	border-color: #101010;
-	color: #dcdccc;
+	background-color: #434443 !important;
+	border-color: #101010 !important;
+	color: #dcdccc !important;
```

Comment:
Hummmmm isn't there a better way than adding all these `!important`?
Removing one is a small victory so adding five...